metadata.json

```diff
@@ -25,7 +25,9 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "16.04"
+        "16.04",
+	"20.04",
```

Comment:
formating, pls :)

Reply:
hmm strange in detiled view it is no the smae ident, on overview page it is 🤔

Reply:
might be space vs tab, gimme a sec